pattern-1/bosh-release/jobs/wso2apim/templates/repository/conf/registry.xml

```diff
@@ -1,335 +1,335 @@
-         and URL to connect to the remote instance (ex:- remote BPS server) if you are planning to use a Remote Human Task. You should
-         provide credentials and URL to connect to the local instance if you simply want to use Work List notifications. -->
-    
-    <!--workList serverURL="local://services/" remote="false">
-        <username>admin</username>
-        <password>admin</password>
-    </workList-->
-    
-    <!-- Sample SCM configuration. Please note that checkOutURL, workingDir, and mountPoint are mandatory for each connection. 
-         The update frequency is in minutes. Setting readOnly would disable checkIn even if the checkInURL is set. The credentials
-         are to connect to the corresponding repository and must be provided if required. -->
-    
-    <!--scm>
-        <connection checkOutURL="scm:svn:http://svn.wso2.org/repos/wso2" workingDir="/valid/filesystem/path" mountPoint="/_system/foo" checkInURL="" readOnly="" updateFrequency="">
-        	<username>admin</username>
-        	<password>admin</password>
-        </connection>
-    </scm-->
-    
-    <!-- This defines the list of scheduled tasks for the registry -->
-    
-    <!--tasks>
-        <task name="TaskName" class="com.foo.TaskClass">
-            <trigger cron="cronExpr"/>
-            <property key="foo" value="FooVal" />
-            <property key="bar" value="BarVal" />
-        </task>
-    </tasks-->
-
-</wso2registry>
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<!--
+  ~ Copyright 2005-2011 WSO2, Inc. (http://wso2.com)
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<wso2registry>
+    <!-- 
+    For details on configuring different config & governance registries see;
+    http://wso2.org/library/tutorials/2010/04/sharing-registry-space-across-multiple-product-instances
+    -->
+
+    <currentDBConfig>wso2registry</currentDBConfig>
+    <readOnly>false</readOnly>
+    <enableCache>true</enableCache>
+    <registryRoot>/</registryRoot>
+
+    <dbConfig name="wso2registry">
+        <dataSource>jdbc/WSO2CarbonDB</dataSource>
+    </dbConfig>
+
+    <dbConfig name="govregistry">
+        <dataSource>jdbc/WSO2REG_DB</dataSource>
+    </dbConfig>
+    <remoteInstance url="https://publisher.apim-wso2.com">
+        <id>gov</id>
+        <cacheId><%=properties.wso2apim.mysql.username %>@jdbc:mysql://<%=properties.wso2apim.mysql.hostname %>:3306/<%=properties.wso2apim.mysql.product_db %></cacheId>
```

Comment:
Shall we add the proper parameter here?

Reply:
fixed https://github.com/wso2/pivotal-cf-apim/pull/52/commits/35c6554ff64c8f44c531e2ed5ac7534cb6a2717d